stock_picking_auto_create_lot/models/stock_move.py

```diff
@@ -22,20 +22,3 @@ def _compute_display_assign_serial(self):
         for move in moves_not_display:
             move.display_assign_serial = False
         return
-
-    # pylint: disable=missing-return
-    def _set_quantities_to_reservation(self):
-        super()._set_quantities_to_reservation()
-        for move in self:
-            if move.state not in ("partially_available", "assigned"):
-                continue
-            if (
-                move.product_id.tracking == "none"
-                or not move.product_id.auto_create_lot
-                or not move.picking_type_id.auto_create_lot
-            ):
-                continue
-            for move_line in move.move_line_ids:
-                if move_line.lot_id:
-                    # Create-backorder wizard would open without this.
-                    move_line.qty_done = move_line.reserved_uom_qty
```

Comment:
Can you rebase this PR so that this part is not removed? 
This module in axls-oca has OCA updates